Automattic/wp-calypso
# PoC: AI domain name suggestions in /start

## Proposed Changes

* This is a PoC that integrates an AI domain name suggestion feature in the signup flow's domain step.
* The background for this is covered in the opportunity described in pe7F0s-1Vj-p2#comment-2628
* A detailed write-up of the feature and demo is in pe7F0s-1Vj-p2.

## Testing Instructions



* Apply D161905-code and make sure public-api.wordpress.com is sandboxed.
* Visit `/start` and click on the AI domain search CTA
* Search for a business name and verify that you get domain name suggestions
* Click on a suggestion domain in the modal, and verify that you are able to add it to cart and complete purchase.
